cirq/ops/raw_types.py

```diff
@@ -460,6 +468,104 @@ def validate_args(self, qubits: Sequence['cirq.Qid']):
         _validate_qid_shape(self, qubits)
 
 
+class Tag:
+    """A wrapper around string that can be used to differentiate operations.
```

Comment:
What's the benefit of wrapping string instead of just using a raw string? Seems like easily removed boilerplate.

Reply:
Removed.